wpiutil/src/main/native/cpp/fs.cpp

```diff
@@ -21,6 +21,7 @@
 #define _WIN32_WINNT 0x0601
 #define _WIN32_IE 0x0800  // MinGW at it again. FIXME: verify if still needed.
 #define WIN32_LEAN_AND_MEAN
+#define kInvalidFile reinterpret_cast<void*>(-1)
```

Comment:
Let's make a different PR for the fs.h/cpp change.  Also, this file should be updated to use the new defined values in fs.h instead of replicating those changes here.

Reply:
This change should go away since the other PR has been merged?